modules/swagger-codegen/src/main/resources/python/model.mustache

```diff
@@ -89,9 +89,10 @@ class {{classname}}(object):
 {{/isEnum}}
 {{^isEnum}}
 {{#hasValidation}}
-
+{{#required}}
         if not {{name}}:
```

Comment:
@vishalsanfran @jdevera FYI. I've filed https://github.com/swagger-api/swagger-codegen/pull/3944 with the fix.